bootstrap/config/kfctl_iap.yaml

```diff
@@ -3,23 +3,46 @@
 # All TODO fields need to be changed following user's input before apply
 # TODO change repo on the fly: set it to local tmp dir containing kubeflow registry
 repo: /path/to/local/tmp/containing/kubeflow
-components:
-  - metacontroller
+packages:
+  - argo
+  - pipeline
+  - common
+  - examples
   - jupyter
+  - katib
+  - mpi-job
+  - pytorch-job
+  - seldon
+  - tf-serving
+  - openvino
+  - tensorboard
+  - tf-training
+  - metacontroller
+  - profiles
+  - application
+  - modeldb
+  - gcp
+components:
+  - pytorch-operator
   - ambassador
+  - openvino
+  - jupyter
+  - notebook-controller
+  - jupyter-web-app
   - centraldashboard
   - tf-job-operator
+  - tensorboard
+  - metacontroller
+  - profiles
+  - notebooks
```

Comment:
We are still working on ```notebooks``` and ```notebook-controller``` components.
We can skip them in this PR and add later once work on that side is done.

Reply:
removed.